moto/resourcegroups/responses.py

```diff
@@ -165,6 +168,9 @@ def update_group(self):
     def update_group_query(self):
         group_name = self._get_param("GroupName")
         resource_query = self._get_param("ResourceQuery")
+        group_arn = self._get_param("Group")
```

Comment:
Can we get a test for the `update_group_query` as well? At the moment it just tests the get_group_query twice

Reply:
Done